alerta/models/alert.py

```diff
@@ -468,19 +468,26 @@ def get_tags(query=None):
 
     @staticmethod
     def housekeeping(expired_threshold=2, info_threshold=12):
-        for (id, event, status, last_receive_id) in db.housekeeping(expired_threshold, info_threshold):
-            if status == 'open':
-                text = 'unshelved after timeout'
-            elif status == 'expired':
-                text = 'expired after timeout'
-            else:
-                text = 'alert timeout status change'
```

Comment:
Please note that I could not see any codepath where text would be set to 'alert timeout status change'. It is therefore omitted from my pull request.